### PR TITLE
add new apps and devices

### LINKED ISSUE #1957: List of useragents not recognized by BrowsCap
Hi, 

Our product BlueConic is using BrowsCap for some time now, and it's working great for us. 
Because we process a lot of requests every day. we are seeing clients which are not recognized by BrowsCap.
I could file a separate request for all of these, but that might not scale well.

I can deliver a CSV file with user agents not recognized, on a regular basis. Please let me know if that is helpful.

Example of such a CSV file is attached.
Thanks in advance and keep up the good work!

[Possible_bots_sept_17.csv.txt](https://github.com/browscap/browscap/files/2387711/Possible_bots_sept_17.csv.txt)

### PR DESCRIPTION
**Pull request by mimmi20**
Fixes #1957.

### CODE REVIEW

File: resources/user-agents/browsers/firefox-on-desktop/firefox-53-on.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
             "Win10_x64", "Win10_64", "Win10_32",
             "Win8_1_x64", "Win8_1_64", "Win8_1_32",
             "Win8_x64", "Win8_64", "Win8_32",
-            "Win7_x64", "Win7_64", "Win7_32"
+            "Win7_x64", "Win7_64", "Win7_32",
+            "Windows"
```
Comment (jaydiablo) [inline resources/user-agents/browsers/firefox-on-desktop/firefox-53-on.json]: What UA was this added for?

File: resources/user-agents/browsers/slimerjs/slimerjs-general.json [hunk 2]
```diff
@@ -5,16 +5,14 @@
   "standard": true,
```
Comment (jaydiablo) [inline resources/user-agents/browsers/slimerjs/slimerjs-general.json]: This file should be named `slimerjs-generic.json` no?

File: resources/user-agents/browsers/firefox-on-desktop/firefox-53-on.json (hunk 1 above)
Comment (mimmi20) [reply resources/user-agents/browsers/firefox-on-desktop/firefox-53-on.json]: testcase `issue-1957-CW`

File: resources/user-agents/browsers/slimerjs/slimerjs-general.json (hunk 2 above)
Comment (mimmi20) [reply resources/user-agents/browsers/slimerjs/slimerjs-general.json]: done

**Pull request merged**